Makefile

```diff
@@ -123,6 +123,7 @@ FFLAGS ?= $(OPT) $(FFLAGS.$(FC_VENDOR))
 
 ifeq ($(COVERAGE), 1)
   CFLAGS += --coverage
+  CXXFLAGS += --coverage
```

Comment:
We could instead change `CFLAGS` to `CPPFLAGS` here instead.

Unrelated: It seems strange to have both `CPPFLAGS` and `CXXFLAGS` running around.

Reply:
`CPPFLAGS` is for the C preprocessor (things like `-I`).
See `make -p -f /dev/null |grep CPPFLAGS`, for example.